src/veh_interact.cpp

```diff
@@ -1826,7 +1826,7 @@ void veh_interact::display_details( const vpart_info *part )
     if ( part->fuel_type != "NULL" ) {
         fold_and_print(w_details, line+4, col_1, ( vertical_menu ? column_width : details_w ), c_white,
                        _("Charge: <color_ltgray>%s</color>"),
-                       part->fuel_type.c_str());
+                       _(part->fuel_type.c_str()));
```

Comment:
`fuel_type` is actually an identifier (an ammo type). In `veh_interact::do_refill`, it's translated like this:
```C++
ammo_name( vehicle_part_types[...].fuel_type ).c_str()
```

Some parts of the code seem to think it's an item type id.